### ISSUE TITLE
Accessibility: [MAS1.1.1]:Ensure that elements have alternate text for image.

### ISSUE THREAD

**Issue by tylerbutler** [microsoft/FluidFramework#3823]
**Build details:**

**OS Version:** 2004(20221.1000)

**Edge Dev Version**: 87.0.654.0 (Official build) dev (64-bit)

**URL:** <https://fluidframework.com/playground/?path=/docs/welcome--page>

**Pre-Requisite:**

Add Accessibility Insights extension for web to Edge Anaheim (url:[https://accessibilityinsights.io/](https://accessibilityinsights.io/))

  

**Repro Steps:**

1\. Open the given URL with valid credentials.

2.Now, Run fastpass using Accessibility Insights for Web and observe the failure instance image alt-text.

  
**Actual Result:**  

**

\<img> element does not have alternate text or role of none or role of presentation

**

**Expected Result:**  

Ensure that \<img> element should have alternate text or role of none or role of presentation

  

**User Impact:**  

Screen readers have no way of translating an image into words that gets read to the user, even if the image only consists of text. As a result, it's necessary for images to have short, descriptive alt-text so screen reader users clearly understand the image's contents and purpose.  

  

**MAS Reference:**

**MAS 1.1.1 - Non-text Content**

**Comment by micahgodbolt** [comment]
Image area is currently set to aria-hidden="true" and the page does not show any issues with the image.

**Comment by micahgodbolt** [comment]
I take it back, focusable elements cannot be inside of aria-hidden containers

**Comment by micahgodbolt** [comment]
This was partially fixed for the landing page at least....but there are other demos with images that need alt text

**Comment by micahgodbolt** [comment]
Double checked, and it seems the rest of the site is actually good on image alt text.
